components/logins/src/migrate_sqlcipher_db.rs

```diff
@@ -246,7 +446,13 @@ fn migrate_logins(
                     // Weren't successful with local login, if we have a mirror we should
                     // attempt to migrate it and flip the `is_overridden` to false
                     if let Some(mirror_login) = &login.mirror_login {
-                        match migrate_mirror_login(&conn, &mirror_login) {
+                        match migrate_mirror_login(
```

Comment:
I would have though `if let Err(err) =...` would work here too (any match with 2 arms and where 1 is empty is suspect)